libs/application/templates/social-insurance-administration/medical-and-rehabilitation-payments/src/lib/dataSchema.ts

```
@@ -0,0 +1,317 @@
+  TaxLevelOptions,
+} from '@island.is/application/templates/social-insurance-administration-core/lib/constants'
+import { errorMessages as coreSIAErrorMessages } from '@island.is/application/templates/social-insurance-administration-core/lib/messages'
+import { formatBankInfo } from '@island.is/application/templates/social-insurance-administration-core/lib/socialInsuranceAdministrationUtils'
+import { parsePhoneNumberFromString } from 'libphonenumber-js'
+import { z } from 'zod'
+import { NOT_APPLICABLE } from '../utils/constants'
+import { errorMessages } from './messages'
+
+const isValidPhoneNumber = (phoneNumber: string) => {
+  const phone = parsePhoneNumberFromString(phoneNumber, 'IS')
+  return phone && phone.isValid()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)